tests/unit/Statement/StatementListTest.php

```diff
@@ -287,11 +280,40 @@ public function testGivenTraversableWithNonStatements_constructorThrowsException
 		new StatementList( $claimsObject );
 	}
 
-	public function testGivenNonTraversable_constructorThrowsException() {
+	public function testGivenNonTraversableOrArgList_constructorThrowsException() {
```

Comment:
This name is a bit odd. Could mean `!( isTraversable || isArgList )` or `( !isTraversable ) || isArgList`. `GivenNull` would be better.